azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/process/LaunchProcessException.java

```diff
@@ -0,0 +1,26 @@
+package bio.terra.workspace.azureDatabaseUtils.process;
+
+/**
+ * Custom exception class for system or internal exceptions. These represent errors that the user
+ * cannot fix.
+ */
+public class LaunchProcessException extends RuntimeException {
```

Comment:
my java naivete is showing... what do we get from this class that we don't get from RuntimeException?